README.md

```diff
@@ -1 +1,105 @@
-# building-nested-hash-readme
+# Building Nested Hashes
+
+## Objectives
+
+In this lesson, we'll be taking a closer look at multidimensional, or nested, hashes. By the end of this lesson, you should be familiar with:
+
+* The difference between single level and nested hashes
+* How to add data to a nested hash
+
+## Nested Hashes
+
+We are used to seeing single level hashes that looks like this:
+
+```ruby
+jon_snow = {
+  name: "Jon",
+  email: "jon_snow@thewall.we"
+}
+```
+However, what if we want to have a more complicated system that can catalog several people. A nested hash will be particularly useful in this case. 
+
+Let's say we have the following nested hash: 
+
+```ruby
+contacts = {
+  "Jon Snow" => {
+    name: "Jon",
+    email: "jon_snow@thewall.we", 
+    favorite_icecream_flavors: ["chocolate", "vanilla"],
+    knows: nil
+  },
+  "Freddy Mercury" => {
+    name: "Freddy",
+    email: "freddy@mercury.com",
+    favorite_icecream_flavors: ["strawberry", "cookie dough", "mint chip"]
+  }
+}
+```
+### Adding Information to Nested Hashes
+
+#### Example 1:
+
+Your good buddy Jon Snow has just tried mint chip ice cream for the first time. He loved it a lot and now you need to add "mint chip" to his list of favorite ice creams. We already know how to access the array of ice cream flavors that constitute the value of the `:favorite_icecream_flavors` key: 
+
+
+```ruby
```

Comment:
For the first example, I would show the two ways of nested hashes. One is what you did with the like ["Jon Snow"][:favorite_icecream_flavors] jam. The other is the slightly more verbose but I think a bit easier to understand for beginners way of extracting one layer at time into variables so

``
jon_snow = contacts["Jon Snow"]
jons_fav_ice_cream = jon_snow[:favorite_icecream_flavors]
```

I'd show them that way, and then say hey this can be easier.